CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### Bug Fixes
+* **@dlr-eoc/map-ol:**
+  - Set urls for vector tile layers after apply style to override them when tilejson is not provided. This is helpful to use `tms` services where no tilejson is available. Then set the url of the sourceId in your style file to `"url": ""` and add `"tiles": []`. This [skips the request](https://github.com/openlayers/ol-mapbox-style/blob/v8.2.1/src/util.js#L109) to tilejson.
```

Comment:
I cannot quite follow along with this sentence.  Would you extend that description a bit?